app/facades/movie_facade.rb

```diff
@@ -12,4 +12,12 @@ def self.search(query)
       Movie.new(movie_data)
     end
   end
+
+  def self.movie_details(movie_id)
```

Comment:
Again - what beautiful logic - lets just end up with a hash of details calling from various endpoints! Impressive!